src/main/java/com/datasonnet/document/MediaTypes.java

```diff
@@ -391,6 +401,7 @@
         APPLICATION_JAVA = new MediaType("application", "x-java-object");
         APPLICATION_CSV = new MediaType("application", "csv");
         UNKNOWN = new MediaType("unknown", "unknown");
+        APPLICATION_YAML = new MediaType("application", "yaml");
```

Comment:
Is this the registered media type? If not can we see what other frameworks are using?

Reply:
Well this is a mixed bag of nuts currently.
Almost every instance I've found, they have both `text` and `application` types. And depending on what the application is, it will be either `yaml` or `x-yaml`
for instance chrome uses `yaml`, Ruby uses `x-yaml`, [this](https://www.freeformatter.com/mime-types-list.html) mime type list uses `yaml`, and [this](https://www.quora.com/What-is-the-correct-MIME-type-for-YAML-documents) response dictates that since its not registered yet, it should be `x-yaml`

Reply:
Just saw that if you upload a .yaml file to S3 it automatically sets its content type to app/x-yaml... so starting to lean a little more towards that, plus it'd be consistent with our app/x-java-object one